test/check-machines-virtualization

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/python3
+
+# This file is part of Cockpit.
+#
+# Copyright (C) 2021 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+
+# import Cockpit's machinery for test VMs and its browser test API
+TEST_DIR = os.path.dirname(__file__)
+sys.path.append(os.path.join(TEST_DIR, "common"))
+sys.path.append(os.path.join(os.path.dirname(TEST_DIR), "bots/machine"))
+
+from machineslib import VirtualMachinesCase  # noqa
+from testlib import nondestructive, test_main, wait  # noqa
+
+
+@nondestructive
+class TestMachinesVirtualization(VirtualMachinesCase):
+
+    def testVirtualizationDisabled(self):
+        b = self.browser
+        m = self.machine
+
+        m.execute("touch /tmp/cpuinfo.txt && mount --bind /tmp/cpuinfo.txt /proc/cpuinfo")
+        self.addCleanup(m.execute, "umount /proc/cpuinfo || true")
+
+        self.login_and_go("/machines")
+
+        b.wait_visible("div.pf-c-empty-state .pf-c-title:contains('Virtualization support is not available')")
+
+        b.click(".pf-c-empty-state__primary .pf-c-button")
+
+        b.wait_visible("#card-pf-storage-pools")
+        b.wait_visible("#card-pf-networks")
+        b.wait_visible("#virtual-machines-listing")
+
+        b.wait_visible(".pf-c-page .pf-c-card__title:contains('Virtual machines')")
+
+
+    def testVirtualizationEnabled(self):
+        b = self.browser
+
```

Comment:
No need for this test, all our other tests do indirectly test this :)

Reply:
Okay :)